Description: replace mocha compiler use by test on compiled files
 This avoid to use --compilers option which is deprecated.
 See https://github.com/mochajs/mocha/wiki/compilers-deprecation
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-03-07

--- a/test/spec/main.coffee
+++ b/test/spec/main.coffee
@@ -2,7 +2,7 @@
 readFileSync = require('fs').readFileSync
 writeFileSync = require('fs').writeFileSync
 IConvLite = require 'iconv-lite'
-ExternalEditor = require('../../src')
+ExternalEditor = require('../../main')
 
 testingInput = 'aAbBcCdDeEfFgG'
 expectedResult = 'aAbBcCdDeE'
